### DISCUSSION TITLE
Unable to Verify Clips - Beta

### DISCUSSION THREAD

**Discussion by esmoyer** [blakeblackshear/frigate#1500] [Ask A Question]
So I decided to give the beta a try to just check out. So far it's been pretty awesome and working well. The only thing I can't seem to figure out is the `"Unable to verify clip for Sidedrive. There were no recordings for this camera."` warning. I'm sure it's something dumb I have overlooked. I have setup the new style of config to get rid of the old Clips and replaced it with the Record and Events. Recording works fine as do the snapshots and events, but I can't seem to figure out the Clips part. It did work for a couple clips but the vast majority just show a snapshot. Wasn't sure if it was complete and working yet, still being in beta. Otherwise, pretty rock solid!

**Comment by esmoyer** [comment]
Feel free to delete or remove this...moving into the beta discussion that was just posted this morning

**Reply by blakeblackshear** [reply]
I fixed it prior to the release. I updated the images previously published, so you can repull.

**Reply by esmoyer** [reply]
Much appreciated...I just got done testing around with the official beta release and it's working as expected now. Excellent work!
